### PR TITLE
Switch to js-cookie

### PR DESCRIPTION
**Pull request by nikolas**
jquery-cookie is deprecated

### CODE REVIEW

File: app/assets/javascripts/js-cookie/js.cookie.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+/*!
+ * JavaScript Cookie v2.0.4
+ * https://github.com/js-cookie/js-cookie
+ *
+ * Copyright 2006, 2015 Klaus Hartl & Fagner Brack
+ * Released under the MIT license
+ */
+(function (factory) {
+	if (typeof define === 'function' && define.amd) {
+		define(factory);
```
Comment (nikolas) [inline app/assets/javascripts/js-cookie/js.cookie.js]: @houndci yeah, that's why the code is checking if `define` is defined, with `typeof`.

**Comment by mcfiredrill** [comment]
@nikolas Nice! Can we add this with rails-assets like we added three.js ?

**Comment by mcfiredrill** [comment]
Man hound-ci is annoying lol.

**Comment by nikolas** [comment]
yea i will add these to rails-assets

**Pull request merged**